### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by Izebiafe**
- used correct git flow.
- followed proper indentation using linter
- followed the figma design to build a responsive portfolio.

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -309,3 +307,110 @@ hr {
...
+  }
+
+  .card {
+    flex-direction: row;
+  }
+
+  .card .card-img {
+    width: 100%;
+    padding-right: 24px;
+  }
```
Comment (brainconnect93) [inline assets/css/style.css]: - [x] Great effort working on your `desktop version` but currently your work cards need some attention and I would like you to stick to the Figma design as much as possible, In the Figma design, the `2nd` and `4th` card has a kind of reverse in content whereby the image goes to the right side and other content is on the left side. Kindly implement this kind of reversed property in your cards too in order to have a perfect replica of the Figma design. 

Figma Design 👇 

org card

File: assets/css/style.css [hunk 2]
```diff
@@ -309,3 +307,110 @@ hr {
...
+  .contact input[type="text"],
+  .contact input[type="email"] {
+    border: 0;
+    border: 1px solid #fff;
+    width: 100%;
+    height: 48px;
+    font-size: 16px;
+    display: flex;
+    padding-left: 15px;
+  }
```
Comment (brainconnect93) [inline assets/css/style.css]: - [x] I would kindly suggest you stick with the Figma design as much as possible when designing your desktop version of the project. Currently, your contact section of the project does not look like what we have in the Figma design. Kindly reduce the width of the form and make it properly aligned.

Possible Implementation 👇 

  - Kindly reduce the width in the  class `.contact input[type="text"], .contact input[type="email"]`, and `.contact textarea` from `100%` to `50%` and add a margin of `0 auto`
  - Also kindly add this class `.contact .contact-text p` in your `media-query` for desktop design and give it the below properties

```CSS
.contact .contact-text p {
    font-style: normal;
    font-weight: 400;
    font-size: 20px;
    line-height: 28px;
    color: #ebebff;
    width: 70%;
    margin: 15px auto;
}
```


- [x] Also i would kindly suggest you give all the buttons some effect as seen in the Figma design `i.e hover, active, enabled` etc.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @Izebiafe, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## To Highlight 👍

- Descriptive Pull Request Title and Summary. ✔️
- Media Query Perfectly used ✔️
- Flexbox and Grid in order to layout the elements ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers... and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/utilites.css [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+.btn-hover:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.btn-hover-2:hover {
+  border: 1px solid #fff;
+  background: #6070ff;
+  color: #fff;
+}
```
Comment (KanzaTahreem) [inline assets/css/utilites.css]: - [x] Well done for implementing the button interactions 🥇🥇 However, I noticed that you missed the `box-shadow` property for `hover` which is `box-shadow: 0px 8px 16px rgba(64, 83, 252, 0.24);` Please add this ✨

File: assets/css/style.css [hunk 4]
```diff
@@ -309,3 +307,122 @@ hr {
...
+
+  .card {
+    flex-direction: row;
+  }
+
+  .card .card-img {
+    width: 100%;
+    padding-right: 24px;
+  }
```
Comment (KanzaTahreem) [inline assets/css/style.css]: - [OPTIONAL] Good job for reversing the cards as suggested by the reviewer 🥇🥇 Please consider fixing the width of the images in project cards to get a design close to the Figma template as much as possible 🤓🔨 One possible implementation can be 👇

```
.card .card-img {
  // other properties
  flex: 1.5;
}

content {
  flex: 2;
}
```

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @Izebiafe 🙋‍♀️

Good job so far implementing the changes suggested by the previous reviewer . **_However, ther are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Good branch name
- ✅ Correct Linters

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Please check the comments under the review._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Efiamotu-1 — APPROVED** [review]
Hi @Izebiafe , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**